Rakefile

```
@@ -74,6 +74,11 @@ namespace :lint do
       sh 'mint run SwiftFormat Sources Tests Example Package.swift --config script/lint/airbnb.swiftformat --lint'
     end
   end
+
+  desc 'Lints podspec files'
```

The podspec is the name of the file, and there's only one of them
```suggestion
  desc 'Lints the CocoaPods podspec'
```